### ISSUE TITLE
using batchnorm in conv2d discard the bias

### ISSUE THREAD

**Issue by fstrub95** [tensorflow/tensorflow#14173]
Hi!
I observed that whenever I applied batch normalization to conv2d, the bias variable are not created!
version : Tensorflow 1.3

```
import tensorflow

image = tf.placeholder(tf.float32, [None, 14, 14, 1024])

tf.contrib.layers.conv2d(image,
                                 num_outputs=128,
                                 kernel_size=[3,3],
                                 normalizer_fn=tf.layers.batch_normalization,
                                 normalizer_params={"training": False, "reuse": False},
                                 activation_fn=tf.nn.relu,
                                 reuse=False,
                                 scope="conv1")
                                 
tf.contrib.layers.conv2d(image,
                                 num_outputs=128,
                                 kernel_size=[3,3],
                                 #normalizer_fn=tf.layers.batch_normalization,
                                 #normalizer_params={"training": False, "reuse": False},
                                 #activation_fn=tf.nn.relu,
                                 reuse=False,
                                 scope="conv2")
                                 
for v in tf.trainable_variables():
	print(v)
```
output:

<tf.Variable 'conv1/weights:0' shape=(3, 3, 1024, 128) dtype=float32_ref>
<tf.Variable 'conv1/batch_normalization/beta:0' shape=(128,) dtype=float32_ref>
<tf.Variable 'conv1/batch_normalization/gamma:0' shape=(128,) dtype=float32_ref>
<tf.Variable 'conv2/weights:0' shape=(3, 3, 1024, 128) dtype=float32_ref>
**<tf.Variable 'conv2/biases:0' shape=(128,) dtype=float32_ref>**

no bias for conv1 !

Question: Is it a bug or a feature :)

[edit] I also observed the same behavior with mlp.

Thank you very much!

Florian

**Comment by fstrub95** [comment]
Nevermind... it is a feature...
Normalization function to use instead of `biases`. If
      `normalizer_fn` is provided then `biases_initializer` and
      `biases_regularizer` are ignored and `biases` are not created nor added.
      default set to None for no normalizer function

**Comment by WaNePr** [comment]
The batch_normalization already included the addition of the bias term.
check out this link https://stackoverflow.com/questions/46256747/can-not-use-both-bias-and-batch-normalization-in-convolution-layers.
